### ISSUE TITLE
[Question] Can I still configure the system to send me an email even if I reached the Alarm threshold?

### ISSUE THREAD

**Issue by donskytech** [thingsboard/thingsboard#9933]
**Component**
 * Rule Engine

**Description**
I paid up for the Cloud Maker Subscription Plan and I have only two devices.  I have just noticed that if I reach the alarm threshold of 200 then no Alarm is created.
Is it still possible to send an email even if I have reached the alarm threshold of 200?
The value of 200 is a little bit low for a month and with using only 2 devices, it still will be reached immediately.

What are the workarounds?  Please I hope to receive answers as this would make or break if Thingsboard is viable for my IoT system.

**Environment**

 * OS:  Windows
 * ThingsBoard: Cloud
 * Browser: Chrome

**Comment by Backdraft007** [comment]
Which 200 do you mean? I can't follow you.

**Comment by devaskim** [comment]
@Backdraft007 It is about alarm API limits for Maker Plant
![Скриншот 04-01-2024 172757](https://github.com/thingsboard/thingsboard/assets/86909129/01235dbb-0345-4238-bb65-062661e874b7)

**Comment by donskytech** [comment]
> @Backdraft007 It is about alarm API limits for Maker Plant ![Скриншот 04-01-2024 172757](https://private-user-images.githubusercontent.com/86909129/294179127-01235dbb-0345-4238-bb65-062661e874b7.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQzNzE5NTAsIm5iZiI6MTcwNDM3MTY1MCwicGF0aCI6Ii84NjkwOTEyOS8yOTQxNzkxMjctMDEyMzVkYmItMDM0NS00MjM4LWJiNjUtMDYyNjYxZTg3NGI3LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA0VDEyMzQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMzNThiMjdmNjExNzNmYjNlODgyZTExZTNiMTMzNDZlYjNhZjA3ODg1ODU4NjA2YzVmNzYxNTVlOTk2NTUzY2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.L6hNCPsG4H2sCUvHF-X0n2Rd7_m1MN8IKEROkVaZdJE)

Yes, this is correct.  I only have two devices on my dashboard, but exceeding this API limit of 200 in a month is quite easy.
I am just looking for alternatives though

**Comment by mykhailokornieiev** [comment]
As the community has not provided assistance on this matter over an extended period, I am closing this ticket due to inactivity. If it is still relevant, feel free to open a new ticket with a full description of your request.
